torchgen/aoti/fallback_ops.py

```diff
@@ -140,7 +140,7 @@
     "aten.randn.default": {},
     "aten.randn.generator": {},
     "aten.randperm.default": {},
-    "aten.repeat_interleave.Tensor": {},
+    "aten.repeat_interleave.Tensor": {"v2": ["dummy_arg_with_default"]},
```

Comment:
just for my understanding: does this mean "dummy_arg_with_default" only exists for "v2" (and later)? If we later have v3, we would assume that  "dummy_arg_with_default" also exist for v3? And I guess this does not support deleting argument?

Reply:
You are right. It doesn't support delete argument, because that will be BC breaking even in python. For "v3", one needs to specify what new arguments are added. Both v2 and v3 C shim will be generated.